frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx

```diff
@@ -13,16 +17,41 @@ export type RevisionsOverviewListProps = {
   service: K8sResourceKind;
 };
 
+const MAX_REVS: number = 3;
+
 const RevisionsOverviewList: React.FC<RevisionsOverviewListProps> = ({ revisions, service }) => {
   const canSetTrafficDistribution = useAccessReview({
     group: ServiceModel.apiGroup,
     resource: ServiceModel.plural,
     namespace: service.metadata.namespace,
     verb: 'update',
   });
+
+  const filteredRevisions = (): K8sResourceKind[] => {
+    if (!revisions || !revisions.length) {
+      return [];
+    }
+    const traffic = service?.status?.traffic;
```

Comment:
nit: can desctructre it above along with `namespace`

Reply:
done